### ISSUE TITLE
Creating my own dataset

### ISSUE THREAD

**Issue by freesinger** [rusty1s/pytorch_geometric#202]
## ❓ Questions & Help

First of all, thanks for your excellent works. I just started my deep learning study with torch_geometric and it's really helpful!

Heres my questions when I am trying to creating my own dataset. First, how can I add `train_maks, val_mask` and `test_mask` attribute to `Data`?  Is `Data.__setitem__()` suitbal for this? Second, in Cora dataset, I get `Data(x=[2708, 1433], edge_index=[2, 10556], y=[2708], train_mask=[2708], val_mask=[2708], test_mask=[2708])` and `data.train_mask.sum()=140`, dose it mean `*_mask` is multi-hot key used to select data for various use(e.g. train)?

Any help will be appreciated!

**Comment by rusty1s** [comment]
Hi,
you can add masks (or any other attributes) by simply assigning it to your data object, *e.g.*, `data.train_mask = torch.tensor([...], dtype=torch.uint8)`. The `*_masks` we use in the Cora datasets are masks in the size of the number of nodes and contain a `1` if the node belongs to the given set (and `0` otherwise).

**Comment by freesinger** [comment]
It works as expectedly, still got something to figure out.
Sorry to bother you with my rookie questions.
Anyway thank you xD
